admin/server/billing.go

```diff
@@ -219,6 +220,17 @@ func (s *Server) CancelBillingSubscription(ctx context.Context, req *adminv1.Can
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
+	err = s.admin.Email.SendSubscriptionCancelled(&email.SubscriptionCancelled{
+		ToEmail:  org.BillingEmail,
+		ToName:   org.Name,
+		OrgName:  org.Name,
+		PlanName: "Team Plan", // TODO: will this ever be different?
```

Comment:
lets just avoid hard coding the name, if really required we may return active sub from the cancel call or pull active subscription and use that to get the plan name.

Reply:
Good point. Lemme pull from plan details.

Reply:
BTW we also have trial extended email now - https://github.com/rilldata/rill/blob/main/runtime/pkg/email/email.go#L504

Reply:
Do we need to have button to upgrade there as well ? Also I think `Welcome to Rill!` can be dropped from the body.